Doc/library/stdtypes.rst

```diff
@@ -3826,9 +3826,11 @@ another set.  The :class:`frozenset` type is immutable and :term:`hashable` ---
 its contents cannot be altered after it is created; it can therefore be used as
 a dictionary key or as an element of another set.
 
-Non-empty sets (not frozensets) can be created by placing a comma-separated list
-of elements within braces, for example: ``{'jack', 'sjoerd'}``, in addition to the
-:class:`set` constructor.
+Non-empty sets (not frozensets) canb be created by several ways:
```

Comment:
I think empty set can't created by set comprehension, `{}` return a dict, I'll add up the constructor.